lib/controllers/birds.js

```diff
@@ -0,0 +1,40 @@
+const { Router } = require('express');
+const { Bird } = require('../models/Bird.js');
+
+module.exports = Router()
+  .get('/', async (req, res) => {
```

Comment:
you'll want all of the routes wrapped in a ```try catch``` so if something goes wrong you have a better sense of what the error was